datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -1881,6 +1881,12 @@ SELECT MIN(c1), MIN(c2) FROM test
 ----
 0 1
 
+query error min/max was called with 2 arguments. It requires only 1.
```

Comment:
Prior to https://github.com/apache/datafusion/pull/12016 this fails with an assert:

```
     Running bin/sqllogictests.rs (target/debug/deps/sqllogictests-baf44314cb754c4d)
Running "aggregate_skip_partial.slt"
Running "aggregate.slt"
Running "aggregates_topk.slt"
thread 'tokio-runtime-worker' panicked at datafusion/functions-aggregate/src/min_max.rs:71:5:
assertion `left == right` failed
  left: 2
 right: 1
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
External error: task 18 panicked
Error: Execution("1 failures")
```